http/src/main/resources/reference.conf

```diff
@@ -59,7 +59,7 @@ gce {
   region = "us-central1"
   # TODO is it safe to use this zone for all GCE VMs?
   zone = "us-central1-a"
-  customGceImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-gce-debian9-1ccc64e"
+  customGceImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-gce-debian9-2020-10-26"
```

Comment:
Doesn't contain https://github.com/DataBiosphere/terra-docker/pull/170, can regenerate the image once those docker images are built+tested.